monitoring/alerts.rb

```diff
@@ -0,0 +1,216 @@
+require "json"
+require "google/cloud/monitoring"
+require "google/cloud/monitoring/v3/alert_policy_service_client"
+require "google/cloud/monitoring/v3/notification_channel_service_client"
+
+def list_alert_policies project_id:
+  # [START monitoring_alert_list_policies]
+  client = Google::Cloud::Monitoring::AlertPolicy.new
+  project_name = Google::Cloud::Monitoring::V3::AlertPolicyServiceClient.project_path project_id
+  policies = client.list_alert_policies project_name
+  policies.each do |policy|
+    p policy
+  end
+  # [END monitoring_alert_list_policies]
+end
+
+# Enable or disable alert policies in a project.
+# @param project_id [String]
+# @param enable [Boolean]
+# @param filter [String] Optional.
+#   Only enable/disable alert policies that match this filter.
+#   https://cloud.google.com/monitoring/api/v3/sorting-and-filtering
+#
+def enable_alert_policies project_id:, enable:, filter: nil
+  # [START monitoring_alert_enable_policies]
+  client = Google::Cloud::Monitoring::AlertPolicy.new
+  project_name = Google::Cloud::Monitoring::V3::AlertPolicyServiceClient.project_path project_id
+  policies = client.list_alert_policies project_name, filter: filter
+
+  policies.each do |policy|
+    if policy.enabled.value == enable
+      p "Policy #{policy.name} is already #{policy.enabled.value ? "enabled" : "disabled"}"
+    else
+      policy.enabled.value = enable
+      update_mask = Google::Protobuf::FieldMask.new paths: ["enabled"]
+      # client.update_alert_policy policy, update_mask
+      p "#{enable ? "Enabled" : "Disabled"} #{policy.name}"
```

Comment:
Nit: could we do `puts` instead of `p` for strings?

Reply:
done